thirdparty/assimp/CREDITS

```diff
@@ -1,183 +0,0 @@
-===============================================================
-Open Asset Import Library (Assimp)
```

Comment:
On the other end the `CREDITS` were a nice touch so that Assimp contributors are properly credited for their work. It makes the "assimp team" from `LICENSE` explicit.

Reply:
I'll definitely re-add this

Reply:
Fixed